### PR TITLE
[draftt-62] Login button signup button goes to signup and signup back works

### PR DESCRIPTION
**Pull request by adil1508**
# What

- Added navigation to Login screen sign up button to go signup page
- Added navigation to Signup screen back arrow key to go back to previous screen
- Added navigation to Signup screen forward arrow key to go to Tester
- Updated Login screen sign up button text color

### CODE REVIEW

File: App/src/screens/SignupScreen.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const SignupScreen = () => {
             </View>
 
             <View style={styles.confirmSignupStyle}>
-                <TouchableOpacity style={styles.arrowButtonStyle}>
+                <TouchableOpacity style={styles.arrowButtonStyle} onPress={() => {navigation.navigate('Login')}}>
```
Comment (aarajh) [inline App/src/screens/SignupScreen.js]: Goes to login not the tester screen.

File: App/src/screens/LoginScreen.js [hunk 2]
```diff
@@ -65,8 +65,8 @@ const LoginScreen = () => {
                     <Text style={{textAlign : 'center'}}>Sign in with Facebook placeholder</Text>
```
Comment (aarajh) [inline App/src/screens/LoginScreen.js]: Maybe a good time to add a basic sign in function that only redirects to the home screen and nothing else?

File: App/src/screens/SignupScreen.js (hunk 1 above)
Comment (adil1508) [reply App/src/screens/SignupScreen.js]: Does now

File: App/src/screens/LoginScreen.js (hunk 2 above)
Comment (adil1508) [reply App/src/screens/LoginScreen.js]: Roger that

Comment (adil1508) [reply App/src/screens/LoginScreen.js]: donezo

**Comment by aarajh** [comment]
Looks good to me

**Pull request merged**